db/data_migration/20150624075140_populate_featured_policies.rb

```diff
@@ -0,0 +1,41 @@
+
+def policies_to_organisations(policies)
+  organisation_content_ids = policies.inject({}) { |memo, policy|
+    orgs = policy["links"]["organisations"]
+    orgs.each { |org| memo[org["content_id"]] = 1 } if orgs
+    memo
+  }.keys
```

Comment:
This was a way of getting a uniq collection of orgs. It turns out ruby can just do that on an array for you though. Seems I have been in JS land for to long. Updated, hopefully its clearer.